include/ck/utility/sequence.hpp

```diff
@@ -606,6 +606,12 @@ struct sequence_map_inverse
                                            SeqMap::Size()>::type;
 };
 
+template <index_t... Xs, index_t... Ys>
+__host__ __device__ constexpr bool operator==(Sequence<Xs...>, Sequence<Ys...>)
+{
+    return (true && ... && (Xs == Ys));
```

Comment:
return (true && ... && (Xs == Ys));

or

return ((Xs == Ys)&&...);

?

Reply:
Not sure what if `Xs` and `Ys` are empty.

Reply:
Found this:
http://eel.is/c++draft/temp.variadic#9
which shows that if they are empty, the folding expression is true for `&&`.

Will make the change.